GATEWAY/src/providers/tilesApi.service.ts

```diff
@@ -12,12 +12,25 @@ export class TilesApi {
   private virtualTiles: VirtualTile[] = [];
   private loginData: LoginData;
   private activeApp: Application;
+  flagThen: boolean = false;
+  flagCatch: boolean = false;
 
   constructor(private http: Http,
               private storage: Storage) {
   }
 
   /**
+   * The following code will mailny be used for getting private parameters
```

Comment:
Ok, I understand what you mean now, but it is unfortunately out of the question, as we should aim for full unit test coverage